sqliter-driver/src/appleMain/kotlin/co/touchlab/sqliter/interop/Platform.kt

```diff
@@ -0,0 +1,7 @@
+package co.touchlab.sqliter.interop
+
+import kotlinx.cinterop.*
+import platform.Foundation.NSString
+import platform.Foundation.create
+
+actual inline fun bytesToString(bv:CPointer<ByteVar>):String = NSString.create(bv).toString()
```

Comment:
There are two overloads taking `CPointer<ByteVar>`, one of which was deprecated. I am not sure which Kotlin is guaranteed to pick, so I'd recommend going explicit with `create(uTF8String = bv)` or `create(bv, NSUTF8StringEncoding)`.

Reply:
Good idea. Will add that.